scout/utils/scout_requests.py

```diff
@@ -28,30 +30,24 @@ def get_request(url):
     """
     try:
         LOG.info("Requesting %s", url)
-        response = urllib.request.urlopen(url, timeout=20)
-        if url.endswith(".gz"):
-            LOG.info("Decompress zipped file")
-            data = gzip.decompress(response.read())  # a `bytes` object
-        else:
-            data = response.read()  # a `bytes` object
-        decoded_data = data.decode("utf-8")
-    except HTTPError as err:
+        response = requests.get(url, timeout=20)
+        if response.status_code != 200:
+            response.raise_for_status()
+        LOG.info("Encoded to %s", response.encoding)
+    except requests.exceptions.HTTPError as err:
         LOG.warning("Something went wrong, perhaps the api key is not valid?")
         raise err
-    except URLError as err:
-        LOG.warning("Something went wrong, are you connected to internet?")
+    except requests.exceptions.MissingSchema as err:
```

Comment:
Did you get this error while testing?

Reply:
This is the error that is raised by requests when url is invalid. Tested here: 
```python
def test_get_request_bad_url():
    """Test functions that accepts an url and returns decoded data from it"""

    # test function with a url that is not valid
    url = "fakeyurl"
    with pytest.raises(requests.exceptions.MissingSchema):
        # function should raise error
        assert scout_requests.get_request(url)
```